apps/website/package.json

```diff
@@ -10,6 +10,8 @@
   "dependencies": {
     "@heroicons/vue": "^1.0.6",
     "ethers": "^5.6.9",
+    "iotex-antenna": "^0.31.0",
```

Comment:
From project rootdir `npm i iotex-antenna --workspace @casimir/website` and commit the resulting root package-lock.json changes as well.

Reply:
https://github.com/consensusnetworks/casimir/tree/feature/ioPay-desktop-integration#scripts-and-dependencies

Reply:
Resolved.